tests/xharness/BCLTestInfo.cs

```diff
@@ -220,6 +222,8 @@ string GetFileList ()
 		{
 			var testName = TestName == "mscorlib" ? "corlib" : TestName;
 			var main_test_sources = Path.Combine (MonoPath, "mcs", "class", testName, testName + "_test.dll.sources");
+			if (!File.Exists (main_test_sources))  // TODO: revert this
+				return "";
```

Comment:
???

Reply:
Same as above, I'm removing all these code. We do not longer have the tests to be built. \o/